pkg/redirectpolicy/manager.go

```diff
@@ -382,7 +386,11 @@ func (rpm *Manager) getAndUpsertPolicySvcConfig(config *LRPConfig) {
 		for i, mapping := range config.frontendMappings {
 			ports[i] = mapping.fePort
 		}
-		ip := rpm.svcCache.GetServiceFrontendIP(*config.serviceID, lb.SVCTypeClusterIP)
+		ip, err := rpm.svcCache.GetServiceFrontendIP(*config.serviceID, lb.SVCTypeClusterIP)
+		if err != nil {
+			log.Debugf("service information not available yet for policy %v", *config)
```

Comment:
Same here

Reply:
This log message is redundant so I've removed it.